user_tools/docs/user-tools-databricks-aws.md

```diff
@@ -0,0 +1,156 @@
+| **gpu_cluster_recommendation** | The type of GPU cluster recommendation to generate. It accepts one of the following (`CLUSTER`, `JOB`, `MATCH`). `MATCH`: keep GPU cluster same number of nodes as CPU cluster; `CLUSTER`: recommend optimal GPU cluster by cost for entire cluster. `JOB`: recommend optimal GPU cluster by cost per job                                                                                                   | `MATCH`                                                                                                                                                                                                                       |     N    |
+| **verbose**                    | True or False to enable verbosity to the wrapper script                                                                                                                                                                                                                                                                                                                                                     | False if `RAPIDS_USER_TOOLS_LOG_DEBUG` is not set                                                                                                                                                                             |     N    |
+| **rapids_options****           | A list of valid [Qualification tool options](../../core/docs/spark-qualification-tool.md#qualification-tool-options). Note that (`output-directory`, `platform`) flags are ignored, and that multiple "spark-property" is not supported.                                                                                                                                                                    | N/A                                                                                                                                                                                                                           |     N    |
+
+#### Use case scenario
+
+A typical workflow to successfully run the `qualification` command in local mode is described as follows:
+
+1. Store the Apache Spark event logs in S3 folder.
+2. A user sets up his development machine:
+   1. configures Java
+   2. installs Databricks CLI and configures the profile and the credentials to make sure the
+      access credentials are stored in the file `~/.databrickscfg` on Unix, Linux, or macOS,
```

Comment:
Same comment about `~/.databrickscfg`. It can be a different file depending on the configurations.

Reply:
Thank you very much! Updated changes.